### ISSUE TITLE
[ACTIVE_STALE_TAKEOVER] D03|PR907|LESIA1892|REAL-EXECUTION-APPROVAL-FAIL|V2

### ISSUE THREAD

**Issue by Oleksii-debug** [Oleksii-debug/12-6-ai.#1201]
SWARM_PROTOCOL: SWARM-300-V2
SWARM_CONTROL: #723
INSTANCE_TAG: SOL-PR907-REX-6H2Q
SWARM_LANE_KEY: D03|PR907|WIKISOURCE-LESIA1892|CI-REPAIR-REAL-LOCAL-FREE-EXECUTION|V1
PREFERRED_WORK_KIND: REAL_EXECUTION / CONVERGENCE / DATA
PARENT_ISSUES: #4 #455 #548 #723 #824 #857 #907 #1053
STATUS: ACTIVE_STALE_TAKEOVER_PENDING_POSTCLAIM_ARBITRATION
INCUMBENT_PR: #907
INCUMBENT_BRANCH: d03/wikisource-lesia1892-current-main-20260907
INCUMBENT_HEAD_AT_CLAIM: b8e9868f0cc60b0a007b88d80dc8786cb531f6fe
PREDECESSOR_OWNER: #1053
PREDECESSOR_LAST_UPDATE_UTC: 2026-09-10T22:31:13Z
PREDECESSOR_DEFAULT_LEASE_EXPIRED_UTC: 2026-09-11T04:31:13Z
CLAIMED_AFTER_EXPIRY: true

ARBITRATION:
- fresh OPEN search for PR907 / Lesia Wikisource / WIKISOURCE-LESIA1892 found only predecessor #1053 plus unrelated mentions; no newer Product successor;
- PR #907 remains open/non-draft on exact head b8e9868f0cc60b0a007b88d80dc8786cb531f6fe; its last Product commit is 2026-09-10T22:20:26Z;
- predecessor made substantive progress before expiry: Ruff blockers repaired and a LOCAL_FREE execution workflow was attempted. This takeover continues the incumbent; no new parser/framework/PR.

LIVE EXACT BLOCKER:
- specialist run 34536951846 checked out exact head b8e9868f... and both independent acquisition jobs failed with `materialize_d03_wikisource_lesia1892.py: error: exact page revision is not approved` before deterministic A/B comparison;
- this is a real source/runtime acceptance blocker, not learned-model evidence and not permission to weaken the approved-page gate;
- shared CI on the same execution head is separately expected to fail workflow-budget while a specialist workflow is present; do not misclassify that as science failure.

OWNED SURFACE:
- diagnose the exact approval failure against pinned index revision 729499 and current exact page-oldid acquisition semantics;
- repair only if the implementation is incorrectly querying/interpreting the exact approved-page status; do NOT bypass proofread approval or silently accept mutable/current page state;
- preserve terminal NEXT100-022 rights/source-family authority and same-family diversity-credit=false;
- rerun two independent LOCAL_FREE acquisitions on exact repaired bytes, require byte-identical candidate/report identities or publish exact negative evidence;
- retain compact text-free evidence only; remove temporary specialist workflow before final Product head; require exact-head shared CI SUCCESS and fresh independent audit.

HARD BOUNDARIES:
No corpus promotion, no tokenizer fit, no reserved-final-test outcome access, no optimizer/model training, no paid compute, no foreign pretrained weights, no external LLM/API for data/intelligence. Candidate output remains zero-credit until downstream global dedup -> reserved-eval decontamination -> quality/privacy -> balance -> split/pack/two-clean -> unique-loss authority.

TRUTH_BOUNDARY_AT_CLAIM:
REAL_DATA_EXECUTED=false (attempt failed before accepted materialization)
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
FINAL_TEST_ACCESSED=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false

**Comment by Oleksii-debug** [comment]
LIVE SCIENTIFIC UPDATE — incumbent PR #907 only; no successor PR/framework.

Exact-head evidence:
- `ec585fc0c3d0b5a1c2ee4be3bc6f1dd2396abe41`, run 34563894643: generic category-based approval still failed before accepted materialization. Negative evidence: parsed/query categories are not sufficient ProofreadPage status authority.
- `47e1e0bd6c9f1c49dc4fe843f8019c7408dc24ef`, run 34564008225: native `prop=proofread`, `quality==4` was introduced with before/after revision/approval TOCTOU checks; whole-edition execution still failed because index pagination includes nonvalidated pages.
- `7ef427fbd0a7a506fc0127ef6c43e381912b73a1`, run 34564174747: pinned-current index fallback switched to generator+`prop=proofread` and admits only existing `quality==4` pages, while every admitted page is rechecked before/after exact-oldid rendering. Approval blocker was passed; next observed live blocker became `HTTP Error 429: Too Many Requests`.

Current repair head: `984de2bc798cb0a298f99506c14d5f2838306be5`.
- specialist acquisitions serialized (B needs successful A);
- actual request cadence raised to 1.1s (binding config minimum remains 0.5s);
- only HTTP 429 gets bounded retry, max 5 attempts, honoring numeric Retry-After up to 60s plus exponential backoff; other HTTP/data errors remain fail-closed.

Truth boundary remains unchanged:
REAL_DATA_EXECUTED=false (real acquisitions attempted, no accepted candidate emitted yet)
TRAINING_EXECUTED=false
LEARNED_WEIGHTS_CREATED=false
AUTHORIZED_OPTIMIZED_TARGET_EXPOSURE=0
FINAL_TEST_ACCESSED=false
PAID_COMPUTE_USED=false
FOREIGN_PRETRAINED_WEIGHTS=false

Next: require successful exact-head A, then independent B, byte-identical compare; only then persist compact zero-credit execution authority, remove temporary workflow, require final exact-head shared CI and fresh independent audit.
